packages/@aws-cdk/aws-iam/lib/policy-document.ts

```diff
@@ -35,11 +47,6 @@ export class PolicyDocument implements cdk.IResolvable {
     this.addStatements(...props.statements || []);
   }
 
-  public resolve(context: cdk.IResolveContext): any {
```

Comment:
It's not a good practice to mix functional changes and code cleanups like moves and formatting since it makes it hard to review the functional change. Please revert. 

If you want to do cleanups, please submit a separate PR for those.